integration_tests/pages/refer/programmeHistoryDetails.ts

```diff
@@ -0,0 +1,71 @@
+import Page from '../page'
+
+export default class ProgrammeHistoryDetailsPage extends Page {
+  constructor() {
+    // Conditional radio buttons add an additional `aria-expanded` field,
+    // so ignore that rule on this page
+    super('Add Accredited Programme history', { accessibilityRules: { 'aria-allowed-attr': { enabled: false } } })
+  }
+
+  shouldContainOutcomeDetailTextArea() {
+    this.shouldContainTextArea('outcomeDetail', 'Can you provide additional detail? (if known)')
+  }
+
+  shouldContainOutcomeOptions() {
+    cy.get('[data-testid="outcome-options"]').within(() => {
+      this.shouldContainRadioItems([
+        { label: 'Complete', value: 'complete' },
+        { label: 'Incomplete', value: 'incomplete' },
+      ])
+    })
+  }
+
+  shouldContainSettingOptions() {
+    cy.get('[data-testid="setting-options"]').within(() => {
+      this.shouldContainRadioItems([
+        { label: 'Community', value: 'community' },
+        { label: 'Custody', value: 'custody' },
+      ])
+    })
+  }
```

Comment:
I wonder `RadioItems` would be clearer than `Options` in these method names? (Same in `SelectProgrammePage`)